go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -95,6 +95,10 @@ func podIPToCookie(podIP net.IP) string {
 
 // AddPod handles the pod add event
 func (n *NodeController) AddPod(pod *kapi.Pod) error {
+	// nothing to do for hostnetwork pod
+	if pod.Spec.HostNetwork {
```

Comment:
Instead of doing it here, can you do it the same way we handle it in the regular code path by putting it in the handler?
https://github.com/ovn-org/ovn-kubernetes/blob/023c09358098a192fcb504e9c8a1694b2754aec5/go-controller/pkg/ovn/ovn.go#L537

Reply:
done